cmd/admin-router.go

```diff
@@ -36,6 +36,8 @@ func registerAdminRouter(mux *router.Router) {
 
 	// Service restart
 	adminRouter.Methods("POST").Queries("service", "").Headers(minioAdminOpHeader, "restart").HandlerFunc(adminAPI.ServiceRestartHandler)
+	// Service update credentials
+	adminRouter.Methods("POST").Queries("service", "").Headers(minioAdminOpHeader, "setCredentials").HandlerFunc(adminAPI.ServiceCredentialsHandler)
```

Comment:
For http methods and header - to be consistent we can use 'set-credentials'  similar to how 'region' is set as 'us-east-1'